### PR TITLE
Added survey focus group meeting recap

### PR DESCRIPTION
**Pull request by kariljordan**
@ErinBecker @tracykteal I invite you both to check out the blog post for the first survey focus group meeting. This is to keep you in the loop. I'd like to merge the post sometime Tuesday (10/23/17).

### CODE REVIEW

**Comment by kariljordan** [comment]
@tracykteal if you have a moment today please review

File: _posts/2017-10-23-survey-focus-group-update.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+Feedback from our instructor community has been that this survey has given some of our learners anxiety about our workshops. Though language around the purpose of the unique identifier was included, some learners feel the questions they are being asked are intrusive, and that the skills-based questions are intimidating. 
+
+Therefore, a call for a survey focus group went out on [discuss](http://lists.software-carpentry.org/pipermail/discuss/2017-October/005486.html) and the [virtual assessment network](https://groups.google.com/a/carpentries.org/forum/#!forum/assessment-network) to take a closer look at this survey, and what changes should be made to make it more inclusive.
+
+Lead by [Kari L. Jordan](https://github.com/kariljordan), the focus group included [Stephen Childs](https://github.com/sechilds), [Alycia Crall](https://twitter.com/alycrall), [Reshama Shaikh](https://twitter.com/reshamas), [Aleksandra Nenadic](https://github.com/anenadic), [Karen Word](http://twitter.com/karen_word), and [Louisa Bells](https://twitter.com/ChemCurator). These individuals represent have expertise in data science assessment and evaluation.
```
Comment (ErinBecker) [inline _posts/2017-10-23-survey-focus-group-update.md]: typo "represent have"

File: _posts/2017-10-23-survey-focus-group-update.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+        ggplot is an R package that is used to build plots from data in a dataframe. If ‘df’ is your dataframe and has 
+        columns x and y, which of the following lines of code will produce a plot of x versus y?
+          A: ggplot <- df
+          B: ggplot(df)
+          C: ggplot(df, aes(x, y))
+          D: ggplot(df, aes(x, y)) + geom_point()
+          E. None of the above will work.
+
+Our goal in including skills-based questions was to be able to show that learners were leaving our workshops having specific skills working with data.
```
Comment (ErinBecker) [inline _posts/2017-10-23-survey-focus-group-update.md]: "was to be able to show that" --> "was to determine whether"

File: _posts/2017-10-23-survey-focus-group-update.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+Feedback from our instructor community has been that this survey has given some of our learners anxiety about our workshops. Though language around the purpose of the unique identifier was included, some learners feel the questions they are being asked are intrusive, and that the skills-based questions are intimidating.
```
Comment (ErinBecker) [inline _posts/2017-10-23-survey-focus-group-update.md]: "our instructor community" --> "some of our instructors" (to make it clear this isn't a widespread problem)

"feel/are" --> "felt/were" (I prefer past tense phrasing here as it help with the narrative of "identifying a problem and fixing it")

File: _posts/2017-10-23-survey-focus-group-update.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
...ers, an idea was to ask learners to provide their personal e-mail address as their unique identifier. Issues were raised around learners not remembering which e-mail address they gave, and that by providing their e-mail address their responses would not be anonymous. We discussed stating in the introductory language that responses would be *confidential*, but not *anonymous* as research staff (Kari Jordan and Erin Becker) have access to the data. There was also the idea of having SurveyMonkey create unique survey links for participants, but this would not be feasible because our survey links are embedded into the [workshop website template](https://github.com/datacarpentry/workshop-template). SurveyMonkey does offer the ability to create [custom variables](https://help.surveymonkey.com/articles/en_US/kb/What-are-custom-variables-and-how-do-I-use-them) however, this will add more manual work for staff and workshop hosts. A graphic was also discussed as an option for unique identifiers, however, we are concerned that learners may not remember what graphic they chose. We are still searching for a solution, so if you have ideas please [get in touch](mailto:kariljordan@carpentries.org).
```
Comment (ErinBecker) [inline _posts/2017-10-23-survey-focus-group-update.md]: For compliance with our IRB terms, survey responses do need to stay anonymous.

File: _posts/2017-10-23-survey-focus-group-update.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+U.S. based workshop participants are asked demographic questions (race/ethnicity, gender, etc.) Collecting this type of information can be challenging, and we want to be sure we are asking demographic questions in an inclusive manner. Thus, instead of asking *what is your gender*, and having learners select from the following, 
+
+          + Female 
+          + Male 
+          + Other 
+          + Prefer not to say 
+
+the *gender* question was changed to a free response question where learners are able to type in how they identify themselves. This will result in more data cleaning, but this can easily be done in OpenRefine.
```
Comment (ErinBecker) [inline _posts/2017-10-23-survey-focus-group-update.md]: In the over 10,000 responses to the SWC pre-workshop survey, only 29 respondents (0.35%) chose "Other". I *strongly* advise against making this question a pure open-response as we're going to get a lot of variability in how people decide to enter the two most popular answers (male and female). For example, for "female" we're going to get "F", "Female", "female", "woman", "f", and probably some variations and typos on those responses.  It is very important that we be able to report our gender distribution for funding and other metrics, so making this more difficult is not recommended.

File: _posts/2017-10-23-survey-focus-group-update.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+Additionally, instead of asking learners to check a box identifying whether or not they are dealing with a disability, we added the following questions:
+
+        **Accessibility** Do you need support for any accessibility requirements that would improve your experience at the
```
Comment (ErinBecker) [inline _posts/2017-10-23-survey-focus-group-update.md]: I like this. Good addition.

File: _posts/2017-10-23-survey-focus-group-update.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+          + Prefer not to say 
+
+the *gender* question was changed to a free response question where learners are able to type in how they identify themselves. This will result in more data cleaning, but this can easily be done in OpenRefine.
+
+Additionally, instead of asking learners to check a box identifying whether or not they are dealing with a disability, we added the following questions:
+
+        **Accessibility** Do you need support for any accessibility requirements that would improve your experience at the      
+        conference? If so, please describe.  
+
+        **Accommodation. Check all that apply.**
```
Comment (ErinBecker) [inline _posts/2017-10-23-survey-focus-group-update.md]: We need to keep in mind that not all of our workshop venues will be able to make all accommodations. For example, providing translation for every language is not feasible. Also, these survey responses go to the workshop instructors, not the host, and the instructors are often not local and would not know how to arrange for these accommodations.

File: _posts/2017-10-23-survey-focus-group-update.md (hunk 5 above)
Comment (tracykteal) [reply _posts/2017-10-23-survey-focus-group-update.md]: Agree we should update this question. However, we shouldn't have it as free-response, as Erin mentioned for doing analysis of the data. There are best practices in asking gender identity questions. Let's find those best practices and use them here.

Comment (kariljordan) [reply _posts/2017-10-23-survey-focus-group-update.md]: I'll update the question as:
To which gender do you most identify:
+ Female
+ Male
+ Transgender female
+ Transgender male
+ Gender variant/non-conforming
+ Not listed
+ Prefer not to answer

File: _posts/2017-10-23-survey-focus-group-update.md (hunk 7 above)
Comment (kariljordan) [reply _posts/2017-10-23-survey-focus-group-update.md]: If this is information we aren't interested in collecting, we will not ask these questions. Instead, we will ask learners to contact their workshop host if they have accessibility or other accommodation needs.

**Comment by kariljordan** [comment]
@tracykteal post has been revised
@ErinBecker welcome to review

I'd like to post this before November 1st if possible.

**Pull request merged**